tests/fixtures/language_repo.sh

```diff
@@ -39,13 +39,14 @@ git commit -q -am c2
 
 #
 # expected output generated using this command in repo directory
-# onefetch --show-logo never --ascii-language <language>
+# onefetch --show-logo never --ascii-language <language> | sed 's/%/%%/g' 
+# REGEX_ANYTHING manually inserted to handle git version numbers on different servers
 #
 
 case "${1}" in
   verilog)
 cat >expected << '__expected_output__'
-[39;1monefetch-committer-name[0m [39;1m~[0m [39;1mgit version 2.37.2[0m
+[39;1monefetch-committer-name[0m [39;1m~[0m [39;1mgit version REGEX_ANYTHING[0m
```

Comment:
So as I understand it the intent is to write the expected value to a file and read that file in the test. TBH I think it would be better if the expected output just existed in the repo, in a file called for example `test_language_repo.stdout`. This would allow the usage of the `include_str!` macro so the test doesn't have to read a file at runtime. Also, replacing `REGEX_ANYTHING` with regex seems a bit hacky IMO.

I wish we could simply assert the exact output, but without a reasonable way to make the git version reproducible across systems (preferably in CI *and* local runs), I guess this can't be done. Perhaps this test should just assert some regex matches to ensure that the expected data exists, and in the expected format.

Reply:
makes sense, in 7b26b88 moves the regex into code, by removing the escape characters the regex becomes more readable too.